js/intercom.js

```diff
@@ -0,0 +1,46 @@
+jQuery(document).ready(function () {
+
+    Quantimodo.getCurrentUser(function (user) {
+        console.debug(user);
```

Comment:
it's defined at any modern browser. seems like only place where it is not defined - your settings